app/src/main/res/layout/fragment_onboarding_page.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="center_horizontal"
+        android:orientation="vertical">
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            android:text="@string/how_it_works" />
+
+        <TextView
+            android:id="@+id/title"
+            style="@style/TextView.ScreenTitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            tools:text="Safe Communities" />
+
+        <ImageView
+            android:id="@+id/image"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/padding_normal"
+            android:src="@drawable/ic_how_it_works_04" />
+
+        <TextView
+            android:id="@+id/subtitle"
+            style="@style/TextView.ScreenSubtitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/padding_normal"
+            tools:text="Jane and Sam help keep their communities safe. They share the app so that others can, too." />
```

Comment:
Replace with `@string`

Reply:
Also is this just acting as a default, to be replaced dynamically with the other values?

Reply:
yep. it is there for development purposes and won't be visible to users